drivers/spi/spi_nrfx_spim.c

```diff
@@ -178,12 +178,20 @@ static void transfer_next_chunk(struct device *dev)
 		xfer.tx_length   = spi_context_tx_buf_on(ctx) ? chunk_len : 0;
 		xfer.p_rx_buffer = ctx->rx_buf;
 		xfer.rx_length   = spi_context_rx_buf_on(ctx) ? chunk_len : 0;
-		result = nrfx_spim_xfer(&dev_config->spim, &xfer, 0);
-		if (result == NRFX_SUCCESS) {
-			return;
-		}
 
-		error = -EIO;
+#if SOC_NRF52832 /* This SPIM driver is only used by the NRF52832 if SOC_NRF52832_PAN_58_OVERRIDE is enabled */
+		if (xfer.rx_length == 1 && xfer.tx_length <= 1) {
+			LOG_DBG("Transaction aborted since it would trigger NRF52832 PAN 58");
```

Comment:
i would increase the level of this log. At least to warning.

Reply:
Excellent suggestion - done.